setup/nuvfile.yml

```diff
@@ -118,4 +124,16 @@ tasks:
         ;; 
         (*) echo nothing installed yet
         esac
+  
+  notify:
+    internal: true
+    silent: true
+    vars:
+      MESSAGE: '{{default 0 .MESSAGE}}'
+    cmds:
+      - |
+        if test -z "$DO_NOT_NOTIFY_NUVOLARIS" 
+        then
+            bash ./scripts/notify.sh "Installed '{{.MESSAGE}}'"
```

Comment:
All tasks should work cross-platforms. Have you tried it on windows without git bash?

Reply:
I've separate the notification logic in another file but to reduce the risks of incompatibility we could use the curl command directly:

`then
    curl -d "{{.MESSAGE}}" https://ntfy.sh/nuvolaris-nir
`

Reply:
Yeah that would be better